### ISSUE TITLE
Improving User Experience for Contributing $ to Sudo Room via Payments or Offline

### ISSUE THREAD

**Issue by wrought** [sudoroom/sudo-humans#42]
Right now, it's definitely do-able to take a single recurring payment based on pre-determined subscription amounts / frequencies via `sudo-humans`. After #41 and #33 are done, we should also be able to add additional "fees" for goods/services/etc that get compiled into a total recurring donation. 

However, this doesn't account for:
- Overall user experience of contributing $$$ to sudo room
- Taking one-time donations / payments
- Taking non-online non-stripe non-bitcoin donations / payments (e.g. cash). 

What do we do???

**Comment by rcsheets** [comment]
I'm going to split this into multiple issues and then close this one, for easier tracking.
